addons/sale/models/payment_transaction.py

```diff
@@ -143,7 +144,9 @@ def _send_invoice(self):
             )
             invoice_to_send.is_move_sent = True # Mark invoice as sent
             for invoice in invoice_to_send.with_user(SUPERUSER_ID):
```

Comment:
the with_user should be moved between the `with_context` and `message_post_with_template` call, like the previous PRs.